src/Compilers/VisualBasic/Portable/Binding/InitializerSemanticModel.vb

```diff
@@ -134,7 +134,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Select
 
             Dim expressionInitializer = TryCast(boundInitializer, BoundExpression)
-            Return If(expressionInitializer IsNot Nothing, New BoundEqualsValue(initializer, expressionInitializer), boundInitializer)
+            If expressionInitializer IsNot Nothing Then
+                If Me.MemberSymbol.Kind = SymbolKind.Parameter Then
+                    Return New BoundParameterEqualsValue(initializer, DirectCast(Me.RootBinder.ContainingMember, ParameterSymbol), expressionInitializer)
```

Comment:
Of right, I forgot to call this out. Preexisting code above gets to the parameter symbol via Me.RootBinder.ContainingMember, and so I copied that. @AlekseyTs, why does the code above do that rather than access Me.MemberSymbol?